airflow/models/serialized_dag.py

```diff
@@ -35,6 +35,7 @@
 from airflow.serialization.serialized_objects import DagDependency, SerializedDAG
 from airflow.settings import COMPRESS_SERIALIZED_DAGS, MIN_SERIALIZED_DAG_UPDATE_INTERVAL, json
 from airflow.utils import timezone
+from airflow.utils.serialization import serialize
```

Comment:
Can we avoid adding new things in to utils namespace? It's already a "dumping ground"

How about


```suggestion
from airflow.serialization import serialize
```

instead?

Reply:
yes we can, but I am trying to avoid confusion for myself at the moment :-).